app/interactors/user.js

```diff
@@ -0,0 +1,18 @@
+const { emailRepeatedError, databaseError } = require('../errors');
+const { findUserByEmail } = require('../services/user');
+const { createUser } = require('../services/user');
+const logger = require('../logger');
+
+exports.createUser = ({ body }, res, next) =>
```

Comment:
the interactor shouldnt deal with anything thats framework or API related, no `request`, `response` or `next`. Just have it be a function that takes in the parameters it needs from the request parameters and give out a response or, alternatively, throw an error and the controller will handle it instead.
Also in this function you make help readability by using async/await, I have no problem if you mix and match both awaits and regular promises